### PR TITLE
Add some problem to Linkedlist

### PR DESCRIPTION
**Pull request by danghai**
Hello,
I add some problem related to Linkedlist. Could you help me review them? Thanks

### CODE REVIEW

File: linkedlist/is_sort.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    while current.next:
+        if current.val > current.next.val:
+            return False
+        current = current.next
+    return True
+
+"""
+Print content and result of a list
+"""
+def print_result(head):
```
Comment (goswami-rahul) [inline linkedlist/is_sort.py]: There is no need for this `print_result` function.  
Writing unittests is enough.

File: linkedlist/loop_detection.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+2. Move FastPointer at a rate of 2 steps, and SlowPointer at a rate of 1 step
+3. When they collide, move SlowPointer to head. Keep FastPointer
+4. Move SlowPointer and FastPointer at a rate of one step. Return collision point.
+"""
+class Node:
+    def __init__(self,x):
+        self.val = x
+        self.next = None
+
+def loop_detection(head):
```
Comment (goswami-rahul) [inline linkedlist/loop_detection.py]: Please remove this file `loop_detection.py`.  
Similiar algorithms have already been done in `linkedlist/is_cyclic.py` and `linkedlist/first_cyclic_node.py`.

File: linkedlist/remove_range.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+class Node:
+    def __init__(self,x):
+        self.val = x
+        self.next = None
+
+def remove_range(head, start, end):
+    assert(start <= end)
+    # Case: remove node at head
+    if start==0:
+        for i in range(0,end+1):
```
Comment (goswami-rahul) [inline linkedlist/remove_range.py]: Please follow PEP8 guide, and put spaces around ` == ` and after `, ` in `range`.

File: linkedlist/remove_range.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+            if head != None:
+                head = head.next
+    else:
+        current = head
+        # Move pointer to start position
+        for i in range(0,start-1):
+            current = current.next
+        # Remove data until the end
+        for i in range(0,end-start+1):
```
Comment (goswami-rahul) [inline linkedlist/remove_range.py]: pep8 fix

File: linkedlist/remove_range.py [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+            if current != None and current.next != None:
+                current.next = current.next.next
+    return head
+
+"""
+Print result of a list
+"""
+def print_result(head):
```
Comment (goswami-rahul) [inline linkedlist/remove_range.py]: no need for `print_result`. unittests will do.

Comment (danghai) [reply linkedlist/remove_range.py]: What do you mean unittests?

File: linkedlist/remove_range.py (hunk 3 above)
Comment (danghai) [reply linkedlist/remove_range.py]: Thanks, I will fix now. 1st using Python

File: linkedlist/remove_range.py (hunk 5 above)
Comment (goswami-rahul) [reply linkedlist/remove_range.py]: In this repo, most files have tests written using `unittest` module. I meant that using `unittest` is preferred over using `print` statements.

**Pull request merged**